Leetcode/intervals.md

```diff
@@ -0,0 +1,78 @@
+# Intervals
```

Comment:
Your Pull Request title: LEETCODE Middle of the linked list solution

Check result:

* Pull Request title must contain group number in ['1021', '1022', '1013', '2021', '2022']
* Pull Request title action must start with ['Added', 'Deleted', 'Refactored', 'Moved', 'Fixed']